delete/wp-site

```diff
@@ -28,18 +31,27 @@ rm -f /etc/nginx/sites-enabled/"$FQDN"
 rm -f /etc/nginx/sites-available/"$FQDN"
 rm -f /etc/php5/fpm/pool.d/"$FQDN".conf
 
-# Revoke certificate. Will try to revoke all certs. 
-for certificate in $(ls $letsencryptFolder/archive/$FQDN | grep "cert"); do
-	echo "Revoking cert $certificate"
-	../letsencrypt/letsencrypt-auto revoke --cert-path "$letsencryptFolder"/archive/"$FQDN"/"$certificate"
-	echo ""
-done
-# Remove config for domain
-rm -f "$letsencryptFolder"/"$FQDN".cli.ini
+
+# Revoke certificate if config for site exists. Will try to revoke all certs. 
+if [ -a $letsencryptFolder/$FQDN.cli.ini ]
+then
+	for certificate in $(ls $letsencryptFolder/archive/$FQDN | grep "cert"); do
+		echo "Revoking cert $certificate"
+		../letsencrypt/letsencrypt-auto revoke --cert-path "$letsencryptFolder"/archive/"$FQDN"/"$certificate"
+		echo ""
+	done
+	
+	# Remove config for domain
+	rm -f "$letsencryptFolder"/"$FQDN".cli.ini
+fi
```

Comment:
There should be some else statement here if only to inform the user that no Let's Encrypt conf was found for the site and that we assume that no certificates need to be revoked.